doc/source/data/getting-started.rst

```diff
@@ -7,16 +7,16 @@ Getting Started
 In this tutorial you will learn how to:
 
 - Create and save a Ray ``Dataset``.
-- How to transform a ``Dataset`` and pass it into other Ray Tasks.
-- How to create a Ray ``DatasetPipeline`` and run transformations on it.
+- Transform a ``Dataset``.
+- Pass a ``Dataset`` to Ray tasks/actors and access the data inside.
 
 .. _ray_datasets_quick_start:
 
 -------------------
 Dataset Quick Start
 -------------------
 
-Ray Datasets implements `Distributed Arrow <https://arrow.apache.org/>`__.
+Ray Datasets implements Distributed `Arrow <https://arrow.apache.org/>`__.
 A Dataset consists of a list of Ray object references to *blocks*.
 Each block holds a set of items in either an `Arrow table <https://arrow.apache.org/docs/python/data.html#tables>`__
 or a Python list (for Arrow incompatible objects).
```

Comment:
This seems no longer true since we support Pandas? @clarkzinzow

Reply:
@jianoaix Correct, great catch! I think this should be
```
Each block holds a set of items in either an `Arrow table <https://arrow.apache.org/docs/python/data.html#tables>`__ (when creating from or transforming to tabular data), a `Pandas DataFrame <https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.html>`__ (when creating from or transforming to Pandas data), or a Python list (otherwise).
```

Reply:
Alright, fixed it.